tools/stubgen/README.md

```diff
@@ -0,0 +1,59 @@
+# Stub generator
+
+This package generates a stub file for classes defined in C++ in Scipp.
+It is specialized to Scipp and will not work with other projects.
+
+## Usage
+
+Simply call the package:
+
+```sh
+python -m stubgen
+```
+
+or use tox:
+
+```shell
+tox -e stubgen
+```
+
+This will build a new stub file by importing and inspecting scipp.
+So you need to make sure that it has built and is importable.
+In tox, this can be achieved by calling `tox -e lib && tox -e editable`.
```

Comment:
Also need to run `conda develop src`, the tox env does not do this currently.

Reply:
Done